coderd/notifications/dispatch/smtp/html.gotmpl

```diff
@@ -8,7 +8,7 @@
   <body style="margin: 0; padding: 0; font-family: -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif; color: #020617; background: #f8fafc;">
     <div style="max-width: 600px; margin: 20px auto; padding: 60px; border: 1px solid #e2e8f0; border-radius: 8px; background-color: #fff; text-align: left; font-size: 14px; line-height: 1.5;">
       <div style="text-align: center;">
-        <img src="https://coder.com/coder-logo-horizontal.png" alt="Coder Logo" style="height: 40px;" />
+        <img src="{{ logo_url }}" alt="{{ app_name }} Logo" style="height: 40px;" />
```

Comment:
note: my immediate thought was to use a [Data URL](https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data) but apparently support for this is extremely limited.